Solutions/vArmour/Workbooks/vArmour_AppContoller_Workbook.json

```diff
@@ -0,0 +1,186 @@
+            }
+          },
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "title": "Top 10 Sources of Violations (Potential Compromised Workloads or User Risk)",
+              "items": [
+                {
+                  "type": 3,
+                  "content": {
+                    "version": "KqlItem/1.0",
+                    "query": "CommonSecurityLog\n| where DeviceVendor == \"vArmour\"\n| where DeviceProduct == \"AC\"\n| where Activity == \"POLICY_VIOLATION\"\n| summarize count() by SourceIP | top 10 by count_ desc \n",
+                    "size": 0,
+                    "timeContext": {
+                      "durationMs": 86400000
+                    },
+                    "queryType": 0,
+                    "resourceType": "microsoft.operationalinsights/workspaces",
+                    "sortBy": []
+                  },
+                  "customWidth": "50",
+                  "name": "query - 2"
+                }
+              ]
+            },
+            "customWidth": "33",
+            "name": "group - 4",
+            "styleSettings": {
+              "showBorder": true
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "CommonSecurityLog\n| where DeviceVendor == \"vArmour\"\n| where DeviceProduct == \"AC\"\n| where Activity == \"POLICY_VIOLATION\"\n| extend PolicyName = extract(\"(\\\\w+{.*})\", 1, DeviceCustomString1)\n| summarize count() by PolicyName | top 10 by count_ desc \n",
+              "size": 0,
+              "title": "Policies with Violations (Risk Identification)",
+              "timeContext": {
+                "durationMs": 86400000
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "customWidth": "33",
+            "name": "query - 4",
+            "styleSettings": {
+              "showBorder": true
+            }
+          }
+        ]
+      },
+      "name": "group - 1"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "union withsource=_TableName *\n| summarize Count=count() by _TableName\n| render barchart",
+              "size": 1,
+              "title": "Types and Number of Events ingested",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "query - 2",
+            "styleSettings": {
+              "showBorder": true
+            }
+          }
+        ]
+      },
+      "name": "group - 3"
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
Workbook should not contain fallbackResourceIds

Reply:
Removed fallbackResourceIds